test-backend.sh

```diff
@@ -20,8 +20,8 @@ export GOFLAGS="-mod=vendor"
 # Invoke ./cover for HTML output
 COVER=${COVER:-"-cover"}
 
-TESTABLE="pkg/auth pkg/proxy pkg/server pkg/helm/actions pkg/helm/handlers"
-FORMATTABLE="${TESTABLE} cmd/bridge pkg/version"
+TESTABLE="./..."
```

Comment:
Perfect, given I just left [this comment](https://github.com/openshift/console/pull/4362#pullrequestreview-361425647)!  👍